language/move-vm/types/src/values/values_impl.rs

```diff
@@ -318,6 +319,8 @@ impl ValueImpl {
             U128(x) => U128(*x),
             Bool(x) => Bool(*x),
             Address(x) => Address(*x),
+            // REVIEW copying resource?
+            Signer(_) => Invalid,
```

Comment:
Good point. Although instead of Invalid you should probably Err as "Invalid" really means "unassigned local". I'd say let's make a real copy for now and keep the comment. I'm planning to review the whole vm story soon and will think about if and how we'll introduce protection for resources in general. (Remember right now resource structs can also be copied.)

Reply:
I agree with Victor on this--allowing the copy seems cleanest given that we do that for structs.